src/averageAges.js

```diff
@@ -37,7 +46,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(
+    person => withChildren
+      ? person.sex === 'f'
+      && people.some(child => child.mother === person.name)
+      : person.sex === 'f'
+  );
```

Comment:
Пропоную записувати в такому форматі
```suggestion
  const filteredMen = people.filter(
     century ? firstCallback : secondCallback
  )